lib/workers/global/config/parse/env.ts

```diff
@@ -237,7 +227,28 @@ export async function getConfig(
     'VSTS_TOKEN',
   ];
 
-  unsupportedEnv.forEach((val) => delete env[val]);
+  for (const val of unsupportedEnv) {
+    delete env[val];
+  }
 
   return config;
 }
+
+async function parseAndValidateOrExit(
+  env: NodeJS.ProcessEnv,
+  configEnvKey: string,
+): Promise<AllConfig> {
+  if (!env[configEnvKey]) {
+    return {};
+  }
+
+  try {
+    const config = JSON5.parse(env[configEnvKey]);
```

Comment:
use our json utils

Reply:
i dont think we can, because the util func catches errors, and here we rely on it to throw when failing to parse it.